stats/stats-server/src/server.rs

```diff
@@ -130,8 +131,9 @@ pub async fn stats(mut settings: Settings) -> Result<(), anyhow::Error> {
         .await?,
     );
 
+    let update_service_cloned = update_service.clone();
     let update_service_handle = tokio::spawn(async move {
-        update_service
+        update_service_cloned
```

Comment:
?
```suggestion
        update_service.clone()
```

Reply:
i'm pretty sure it will try to move `update_service`, resulting in a compile error

Reply:
see the similar pattern when spawning group updaters. ugly but it's rust so borrow checker must be happy :)

Reply:
yep, exactly
<img width="809" alt="image" src="https://github.com/user-attachments/assets/ec2a93c9-3106-41eb-a862-114952b4d707" />